### DISCUSSION TITLE
so-setup command not found

### DISCUSSION THREAD

**Discussion by nicholaswkc34** [Security-Onion-Solutions/securityonion#11652] [2.4] [answered]
### Version

2.4.20

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

other (please provide detail below)

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

16

### Storage for /

300GB

### Storage for /nsm

300 GB

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Dear all forumers, I had installed SO 2.4.20 but after reboot the so-setup command not found when login and when i type so-setup. I don't know where to look for. Please help. Thanks. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by nicholaswkc34** [comment]
I downloaded the latest ISO and only can access the setup after login, not able to launch via command manually with so-setup.

**Comment by cm-ops** [comment]
Are you in the correct directory? `/home/user/Securityonion/setup/`

**Comment by nicholaswkc34** [comment, marked as answer]
Need to add   ./so-setup then it is available.
